### DISCUSSION TITLE
로그는 어느 수준까지 찍어야 할가요?

### DISCUSSION THREAD

**Discussion by LeeKing2017** [jang-199/DoranDoran#210] [Q&A]
메소드가 실행되면 로그를 찍어서 나중에 예기치 못한 에러가 발생했을 때를 대비하려고 합니다.
이를 위해서는 로그를 어느 수준까지 찍어야 대비할 수 있을까요?

**Comment by jang-199** [comment]
info 레벨부터 기록하는 것이 좋아보입니다.^&^
